cmd/minikube/cmd/start_flags.go

```diff
@@ -152,6 +153,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().Bool(deleteOnFailure, false, "If set, delete the current cluster if start fails and try again. Defaults to false.")
 	startCmd.Flags().Bool(forceSystemd, false, "If set, force the container runtime to use sytemd as cgroup manager. Currently available for docker and crio. Defaults to false.")
 	startCmd.Flags().StringVarP(&outputFormat, "output", "o", "text", "Format to print stdout in. Options include: [text,json]")
+	startCmd.Flags().StringP(trace, "", "", "Send trace events. Options include: [gcp]")
```

Comment:
how about adding a check in the validate flags that if the option is not gcp, yell at them and say the valid options are ...

Reply:
I added the valid options to the error message that will be displayed, this is what it would look like:

```
$ minikube start --trace dne

❌  Exiting due to MK_USAGE: error initializing tracing: getting tracer: dne is not a valid tracer, valid tracers include: [gcp]
```